2014-08-18 10:08:25,333 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 11:34:24,854 ERROR [Mina] - A0000038C1903EBV3:both idle timeout!
2014-08-18 11:34:24,854 ERROR [Mina] - A0000038C1903EBV3 is offline!
2014-08-18 13:03:29,842 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 13:15:07,940 ERROR [Mina] - A0000038C1903EBV3 is offline!
2014-08-18 13:15:07,940 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 13:37:07,671 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 13:37:07,671 ERROR [Mina] - A0000038C1903EBV3 is offline!
2014-08-18 13:48:30,714 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 13:48:30,714 ERROR [Mina] - A0000038C1903EBV3 is offline!
2014-08-18 13:52:19,645 ERROR [Mina] - New Connect VIN:A0000038C194C94V3
2014-08-18 14:24:18,842 ERROR [Mina] - New Connect VIN:
2014-08-18 14:24:18,904 ERROR [org.hibernate.util.JDBCExceptionReporter] - ORA-01400: cannot insert NULL into ("SFCV"."STATUS"."VIN")
ORA-06512: at "SFCV.J_PROCESSINFO", line 19
ORA-01403: no data found
ORA-06512: at line 1

2014-08-18 14:24:18,904 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.Exception: org.hibernate.exception.ConstraintViolationException: could not execute query
	at com.fcv.vms.model.HibernateModel.OnProcessReg(HibernateModel.java:242)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:96)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.hibernate.exception.ConstraintViolationException: could not execute query
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:94)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)
	at org.hibernate.loader.Loader.doList(Loader.java:2231)
	at org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2125)
	at org.hibernate.loader.Loader.list(Loader.java:2120)
	at org.hibernate.loader.custom.CustomLoader.list(CustomLoader.java:312)
	at org.hibernate.impl.SessionImpl.listCustomQuery(SessionImpl.java:1722)
	at org.hibernate.impl.AbstractSessionImpl.list(AbstractSessionImpl.java:165)
	at org.hibernate.impl.SQLQueryImpl.list(SQLQueryImpl.java:175)
	at com.fcv.vms.model.HibernateModel.OnProcessReg(HibernateModel.java:234)
	... 11 more
Caused by: java.sql.SQLException: ORA-01400: cannot insert NULL into ("SFCV"."STATUS"."VIN")
ORA-06512: at "SFCV.J_PROCESSINFO", line 19
ORA-01403: no data found
ORA-06512: at line 1

	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:331)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:288)
	at oracle.jdbc.driver.T4C8Oall.receive(T4C8Oall.java:745)
	at oracle.jdbc.driver.T4CCallableStatement.doOall8(T4CCallableStatement.java:215)
	at oracle.jdbc.driver.T4CCallableStatement.executeForRows(T4CCallableStatement.java:965)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1170)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3339)
	at oracle.jdbc.driver.OraclePreparedStatement.execute(OraclePreparedStatement.java:3445)
	at oracle.jdbc.driver.OracleCallableStatement.execute(OracleCallableStatement.java:4394)
	at com.mchange.v2.c3p0.impl.NewProxyCallableStatement.execute(NewProxyCallableStatement.java:3044)
	at org.hibernate.dialect.Oracle9Dialect.getResultSet(Oracle9Dialect.java:319)
	at org.hibernate.jdbc.AbstractBatcher.getResultSet(AbstractBatcher.java:215)
	at org.hibernate.loader.Loader.getResultSet(Loader.java:1805)
	at org.hibernate.loader.Loader.doQuery(Loader.java:697)
	at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:259)
	at org.hibernate.loader.Loader.doList(Loader.java:2228)
	... 18 more
2014-08-18 14:24:18,904 ERROR [Mina] - java.lang.Exception: org.hibernate.exception.ConstraintViolationException: could not execute query exception!
2014-08-18 14:24:19,997 ERROR [Mina] - New Connect VIN:
2014-08-18 14:24:20,013 ERROR [org.hibernate.util.JDBCExceptionReporter] - ORA-01400: cannot insert NULL into ("SFCV"."STATUS"."VIN")
ORA-06512: at "SFCV.J_PROCESSINFO", line 19
ORA-01403: no data found
ORA-06512: at line 1

2014-08-18 14:24:20,013 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.Exception: org.hibernate.exception.ConstraintViolationException: could not execute query
	at com.fcv.vms.model.HibernateModel.OnProcessReg(HibernateModel.java:242)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:96)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.hibernate.exception.ConstraintViolationException: could not execute query
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:94)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)
	at org.hibernate.loader.Loader.doList(Loader.java:2231)
	at org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2125)
	at org.hibernate.loader.Loader.list(Loader.java:2120)
	at org.hibernate.loader.custom.CustomLoader.list(CustomLoader.java:312)
	at org.hibernate.impl.SessionImpl.listCustomQuery(SessionImpl.java:1722)
	at org.hibernate.impl.AbstractSessionImpl.list(AbstractSessionImpl.java:165)
	at org.hibernate.impl.SQLQueryImpl.list(SQLQueryImpl.java:175)
	at com.fcv.vms.model.HibernateModel.OnProcessReg(HibernateModel.java:234)
	... 11 more
Caused by: java.sql.SQLException: ORA-01400: cannot insert NULL into ("SFCV"."STATUS"."VIN")
ORA-06512: at "SFCV.J_PROCESSINFO", line 19
ORA-01403: no data found
ORA-06512: at line 1

	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:331)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:288)
	at oracle.jdbc.driver.T4C8Oall.receive(T4C8Oall.java:745)
	at oracle.jdbc.driver.T4CCallableStatement.doOall8(T4CCallableStatement.java:215)
	at oracle.jdbc.driver.T4CCallableStatement.executeForRows(T4CCallableStatement.java:965)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1170)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3339)
	at oracle.jdbc.driver.OraclePreparedStatement.execute(OraclePreparedStatement.java:3445)
	at oracle.jdbc.driver.OracleCallableStatement.execute(OracleCallableStatement.java:4394)
	at com.mchange.v2.c3p0.impl.NewProxyCallableStatement.execute(NewProxyCallableStatement.java:3044)
	at org.hibernate.dialect.Oracle9Dialect.getResultSet(Oracle9Dialect.java:319)
	at org.hibernate.jdbc.AbstractBatcher.getResultSet(AbstractBatcher.java:215)
	at org.hibernate.loader.Loader.getResultSet(Loader.java:1805)
	at org.hibernate.loader.Loader.doQuery(Loader.java:697)
	at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:259)
	at org.hibernate.loader.Loader.doList(Loader.java:2228)
	... 18 more
2014-08-18 14:24:20,013 ERROR [Mina] -  is offline!
2014-08-18 14:24:20,013 ERROR [Mina] - java.lang.Exception: org.hibernate.exception.ConstraintViolationException: could not execute query exception!
2014-08-18 14:24:28,361 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 14:24:30,952 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:24:30,952 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:24:32,091 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:24:32,091 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:24:32,716 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:24:32,716 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:24:35,291 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:24:35,307 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:24:37,493 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:24:37,509 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:24:41,896 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:24:41,912 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:24:45,253 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:24:45,269 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:24:47,424 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:24:47,439 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:24:51,795 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:24:51,810 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:24:55,105 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:24:55,120 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:25:00,570 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:25:00,585 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:25:01,694 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:25:01,710 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:25:05,004 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:25:05,020 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:25:08,314 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:25:08,330 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:25:10,500 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:25:10,516 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:25:14,919 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:25:14,935 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:25:18,214 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:25:18,229 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:25:20,462 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:25:20,478 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:25:24,802 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:25:24,818 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:25:28,097 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:25:28,112 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:25:33,593 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:25:33,609 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:25:34,702 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:25:34,717 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:25:37,996 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:25:38,012 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:25:41,291 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:25:41,306 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:25:43,555 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:25:43,570 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:25:47,896 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:25:47,911 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:25:51,299 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:25:51,315 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:25:53,392 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:25:53,407 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:25:57,811 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:25:57,826 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:26:01,090 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:26:01,105 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:26:06,617 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:26:06,633 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:26:07,694 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:26:07,710 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:26:11,005 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:26:11,020 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:26:14,299 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:26:14,315 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:26:16,501 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:26:16,516 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:41:42,625 ERROR [Mina] - New Connect VIN:
2014-08-18 14:41:42,687 ERROR [org.hibernate.util.JDBCExceptionReporter] - ORA-01400: cannot insert NULL into ("SFCV"."STATUS"."VIN")
ORA-06512: at "SFCV.J_PROCESSINFO", line 19
ORA-01403: no data found
ORA-06512: at line 1

2014-08-18 14:41:42,687 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.Exception: org.hibernate.exception.ConstraintViolationException: could not execute query
	at com.fcv.vms.model.HibernateModel.OnProcessReg(HibernateModel.java:242)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:96)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.hibernate.exception.ConstraintViolationException: could not execute query
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:94)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)
	at org.hibernate.loader.Loader.doList(Loader.java:2231)
	at org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2125)
	at org.hibernate.loader.Loader.list(Loader.java:2120)
	at org.hibernate.loader.custom.CustomLoader.list(CustomLoader.java:312)
	at org.hibernate.impl.SessionImpl.listCustomQuery(SessionImpl.java:1722)
	at org.hibernate.impl.AbstractSessionImpl.list(AbstractSessionImpl.java:165)
	at org.hibernate.impl.SQLQueryImpl.list(SQLQueryImpl.java:175)
	at com.fcv.vms.model.HibernateModel.OnProcessReg(HibernateModel.java:234)
	... 11 more
Caused by: java.sql.SQLException: ORA-01400: cannot insert NULL into ("SFCV"."STATUS"."VIN")
ORA-06512: at "SFCV.J_PROCESSINFO", line 19
ORA-01403: no data found
ORA-06512: at line 1

	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:331)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:288)
	at oracle.jdbc.driver.T4C8Oall.receive(T4C8Oall.java:745)
	at oracle.jdbc.driver.T4CCallableStatement.doOall8(T4CCallableStatement.java:215)
	at oracle.jdbc.driver.T4CCallableStatement.executeForRows(T4CCallableStatement.java:965)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1170)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3339)
	at oracle.jdbc.driver.OraclePreparedStatement.execute(OraclePreparedStatement.java:3445)
	at oracle.jdbc.driver.OracleCallableStatement.execute(OracleCallableStatement.java:4394)
	at com.mchange.v2.c3p0.impl.NewProxyCallableStatement.execute(NewProxyCallableStatement.java:3044)
	at org.hibernate.dialect.Oracle9Dialect.getResultSet(Oracle9Dialect.java:319)
	at org.hibernate.jdbc.AbstractBatcher.getResultSet(AbstractBatcher.java:215)
	at org.hibernate.loader.Loader.getResultSet(Loader.java:1805)
	at org.hibernate.loader.Loader.doQuery(Loader.java:697)
	at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:259)
	at org.hibernate.loader.Loader.doList(Loader.java:2228)
	... 18 more
2014-08-18 14:41:42,703 ERROR [Mina] - java.lang.Exception: org.hibernate.exception.ConstraintViolationException: could not execute query exception!
2014-08-18 14:41:43,795 ERROR [Mina] - New Connect VIN:
2014-08-18 14:41:43,795 ERROR [org.hibernate.util.JDBCExceptionReporter] - ORA-01400: cannot insert NULL into ("SFCV"."STATUS"."VIN")
ORA-06512: at "SFCV.J_PROCESSINFO", line 19
ORA-01403: no data found
ORA-06512: at line 1

2014-08-18 14:41:43,795 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.Exception: org.hibernate.exception.ConstraintViolationException: could not execute query
	at com.fcv.vms.model.HibernateModel.OnProcessReg(HibernateModel.java:242)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:96)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.hibernate.exception.ConstraintViolationException: could not execute query
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:94)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)
	at org.hibernate.loader.Loader.doList(Loader.java:2231)
	at org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2125)
	at org.hibernate.loader.Loader.list(Loader.java:2120)
	at org.hibernate.loader.custom.CustomLoader.list(CustomLoader.java:312)
	at org.hibernate.impl.SessionImpl.listCustomQuery(SessionImpl.java:1722)
	at org.hibernate.impl.AbstractSessionImpl.list(AbstractSessionImpl.java:165)
	at org.hibernate.impl.SQLQueryImpl.list(SQLQueryImpl.java:175)
	at com.fcv.vms.model.HibernateModel.OnProcessReg(HibernateModel.java:234)
	... 11 more
Caused by: java.sql.SQLException: ORA-01400: cannot insert NULL into ("SFCV"."STATUS"."VIN")
ORA-06512: at "SFCV.J_PROCESSINFO", line 19
ORA-01403: no data found
ORA-06512: at line 1

	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:331)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:288)
	at oracle.jdbc.driver.T4C8Oall.receive(T4C8Oall.java:745)
	at oracle.jdbc.driver.T4CCallableStatement.doOall8(T4CCallableStatement.java:215)
	at oracle.jdbc.driver.T4CCallableStatement.executeForRows(T4CCallableStatement.java:965)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1170)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3339)
	at oracle.jdbc.driver.OraclePreparedStatement.execute(OraclePreparedStatement.java:3445)
	at oracle.jdbc.driver.OracleCallableStatement.execute(OracleCallableStatement.java:4394)
	at com.mchange.v2.c3p0.impl.NewProxyCallableStatement.execute(NewProxyCallableStatement.java:3044)
	at org.hibernate.dialect.Oracle9Dialect.getResultSet(Oracle9Dialect.java:319)
	at org.hibernate.jdbc.AbstractBatcher.getResultSet(AbstractBatcher.java:215)
	at org.hibernate.loader.Loader.getResultSet(Loader.java:1805)
	at org.hibernate.loader.Loader.doQuery(Loader.java:697)
	at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:259)
	at org.hibernate.loader.Loader.doList(Loader.java:2228)
	... 18 more
2014-08-18 14:41:43,795 ERROR [Mina] -  is offline!
2014-08-18 14:41:43,795 ERROR [Mina] - java.lang.Exception: org.hibernate.exception.ConstraintViolationException: could not execute query exception!
2014-08-18 14:41:52,205 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 14:41:53,782 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:41:53,782 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:41:55,156 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:41:55,156 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:41:56,827 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:41:56,827 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:41:59,199 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:41:59,215 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:42:01,619 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:42:01,635 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:42:05,834 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:42:05,834 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:46:09,556 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 14:46:11,024 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:46:11,039 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:46:12,429 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:46:12,429 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:46:16,909 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:46:16,925 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:46:18,002 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:46:18,018 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:46:21,312 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:46:21,328 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:46:24,654 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:46:24,669 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:46:26,809 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:46:26,824 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:46:31,212 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:46:31,227 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:46:34,506 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:46:34,522 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:49:11,947 ERROR [Mina] - New Connect VIN:
2014-08-18 14:49:11,994 ERROR [org.hibernate.util.JDBCExceptionReporter] - ORA-01400: cannot insert NULL into ("SFCV"."STATUS"."VIN")
ORA-06512: at "SFCV.J_PROCESSINFO", line 19
ORA-01403: no data found
ORA-06512: at line 1

2014-08-18 14:49:12,009 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.Exception: org.hibernate.exception.ConstraintViolationException: could not execute query
	at com.fcv.vms.model.HibernateModel.OnProcessReg(HibernateModel.java:242)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:96)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.hibernate.exception.ConstraintViolationException: could not execute query
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:94)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)
	at org.hibernate.loader.Loader.doList(Loader.java:2231)
	at org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2125)
	at org.hibernate.loader.Loader.list(Loader.java:2120)
	at org.hibernate.loader.custom.CustomLoader.list(CustomLoader.java:312)
	at org.hibernate.impl.SessionImpl.listCustomQuery(SessionImpl.java:1722)
	at org.hibernate.impl.AbstractSessionImpl.list(AbstractSessionImpl.java:165)
	at org.hibernate.impl.SQLQueryImpl.list(SQLQueryImpl.java:175)
	at com.fcv.vms.model.HibernateModel.OnProcessReg(HibernateModel.java:234)
	... 11 more
Caused by: java.sql.SQLException: ORA-01400: cannot insert NULL into ("SFCV"."STATUS"."VIN")
ORA-06512: at "SFCV.J_PROCESSINFO", line 19
ORA-01403: no data found
ORA-06512: at line 1

	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:331)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:288)
	at oracle.jdbc.driver.T4C8Oall.receive(T4C8Oall.java:745)
	at oracle.jdbc.driver.T4CCallableStatement.doOall8(T4CCallableStatement.java:215)
	at oracle.jdbc.driver.T4CCallableStatement.executeForRows(T4CCallableStatement.java:965)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1170)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3339)
	at oracle.jdbc.driver.OraclePreparedStatement.execute(OraclePreparedStatement.java:3445)
	at oracle.jdbc.driver.OracleCallableStatement.execute(OracleCallableStatement.java:4394)
	at com.mchange.v2.c3p0.impl.NewProxyCallableStatement.execute(NewProxyCallableStatement.java:3044)
	at org.hibernate.dialect.Oracle9Dialect.getResultSet(Oracle9Dialect.java:319)
	at org.hibernate.jdbc.AbstractBatcher.getResultSet(AbstractBatcher.java:215)
	at org.hibernate.loader.Loader.getResultSet(Loader.java:1805)
	at org.hibernate.loader.Loader.doQuery(Loader.java:697)
	at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:259)
	at org.hibernate.loader.Loader.doList(Loader.java:2228)
	... 18 more
2014-08-18 14:49:12,009 ERROR [Mina] - java.lang.Exception: org.hibernate.exception.ConstraintViolationException: could not execute query exception!
2014-08-18 14:49:13,414 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 14:49:13,805 ERROR [Mina] - A0000038C1903EBV3 is offline!
2014-08-18 14:49:20,950 ERROR [Mina] - New Connect VIN:
2014-08-18 14:49:20,966 ERROR [org.hibernate.util.JDBCExceptionReporter] - ORA-01400: cannot insert NULL into ("SFCV"."STATUS"."VIN")
ORA-06512: at "SFCV.J_PROCESSINFO", line 19
ORA-01403: no data found
ORA-06512: at line 1

2014-08-18 14:49:20,982 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.Exception: org.hibernate.exception.ConstraintViolationException: could not execute query
	at com.fcv.vms.model.HibernateModel.OnProcessReg(HibernateModel.java:242)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:96)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.hibernate.exception.ConstraintViolationException: could not execute query
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:94)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)
	at org.hibernate.loader.Loader.doList(Loader.java:2231)
	at org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2125)
	at org.hibernate.loader.Loader.list(Loader.java:2120)
	at org.hibernate.loader.custom.CustomLoader.list(CustomLoader.java:312)
	at org.hibernate.impl.SessionImpl.listCustomQuery(SessionImpl.java:1722)
	at org.hibernate.impl.AbstractSessionImpl.list(AbstractSessionImpl.java:165)
	at org.hibernate.impl.SQLQueryImpl.list(SQLQueryImpl.java:175)
	at com.fcv.vms.model.HibernateModel.OnProcessReg(HibernateModel.java:234)
	... 11 more
Caused by: java.sql.SQLException: ORA-01400: cannot insert NULL into ("SFCV"."STATUS"."VIN")
ORA-06512: at "SFCV.J_PROCESSINFO", line 19
ORA-01403: no data found
ORA-06512: at line 1

	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:331)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:288)
	at oracle.jdbc.driver.T4C8Oall.receive(T4C8Oall.java:745)
	at oracle.jdbc.driver.T4CCallableStatement.doOall8(T4CCallableStatement.java:215)
	at oracle.jdbc.driver.T4CCallableStatement.executeForRows(T4CCallableStatement.java:965)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1170)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3339)
	at oracle.jdbc.driver.OraclePreparedStatement.execute(OraclePreparedStatement.java:3445)
	at oracle.jdbc.driver.OracleCallableStatement.execute(OracleCallableStatement.java:4394)
	at com.mchange.v2.c3p0.impl.NewProxyCallableStatement.execute(NewProxyCallableStatement.java:3044)
	at org.hibernate.dialect.Oracle9Dialect.getResultSet(Oracle9Dialect.java:319)
	at org.hibernate.jdbc.AbstractBatcher.getResultSet(AbstractBatcher.java:215)
	at org.hibernate.loader.Loader.getResultSet(Loader.java:1805)
	at org.hibernate.loader.Loader.doQuery(Loader.java:697)
	at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:259)
	at org.hibernate.loader.Loader.doList(Loader.java:2228)
	... 18 more
2014-08-18 14:49:20,982 ERROR [Mina] - java.lang.Exception: org.hibernate.exception.ConstraintViolationException: could not execute query exception!
2014-08-18 14:49:23,385 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 14:49:23,557 ERROR [Mina] - A0000038C1903EBV3 is offline!
2014-08-18 14:49:31,826 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 14:49:33,230 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:49:33,230 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:49:35,447 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:49:35,447 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:50:11,503 ERROR [Mina] - New Connect VIN:
2014-08-18 14:50:11,503 ERROR [org.hibernate.util.JDBCExceptionReporter] - ORA-01400: cannot insert NULL into ("SFCV"."STATUS"."VIN")
ORA-06512: at "SFCV.J_PROCESSINFO", line 19
ORA-01403: no data found
ORA-06512: at line 1

2014-08-18 14:50:11,503 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.Exception: org.hibernate.exception.ConstraintViolationException: could not execute query
	at com.fcv.vms.model.HibernateModel.OnProcessReg(HibernateModel.java:242)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:96)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.hibernate.exception.ConstraintViolationException: could not execute query
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:94)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)
	at org.hibernate.loader.Loader.doList(Loader.java:2231)
	at org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2125)
	at org.hibernate.loader.Loader.list(Loader.java:2120)
	at org.hibernate.loader.custom.CustomLoader.list(CustomLoader.java:312)
	at org.hibernate.impl.SessionImpl.listCustomQuery(SessionImpl.java:1722)
	at org.hibernate.impl.AbstractSessionImpl.list(AbstractSessionImpl.java:165)
	at org.hibernate.impl.SQLQueryImpl.list(SQLQueryImpl.java:175)
	at com.fcv.vms.model.HibernateModel.OnProcessReg(HibernateModel.java:234)
	... 11 more
Caused by: java.sql.SQLException: ORA-01400: cannot insert NULL into ("SFCV"."STATUS"."VIN")
ORA-06512: at "SFCV.J_PROCESSINFO", line 19
ORA-01403: no data found
ORA-06512: at line 1

	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:331)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:288)
	at oracle.jdbc.driver.T4C8Oall.receive(T4C8Oall.java:745)
	at oracle.jdbc.driver.T4CCallableStatement.doOall8(T4CCallableStatement.java:215)
	at oracle.jdbc.driver.T4CCallableStatement.executeForRows(T4CCallableStatement.java:965)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1170)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3339)
	at oracle.jdbc.driver.OraclePreparedStatement.execute(OraclePreparedStatement.java:3445)
	at oracle.jdbc.driver.OracleCallableStatement.execute(OracleCallableStatement.java:4394)
	at com.mchange.v2.c3p0.impl.NewProxyCallableStatement.execute(NewProxyCallableStatement.java:3044)
	at org.hibernate.dialect.Oracle9Dialect.getResultSet(Oracle9Dialect.java:319)
	at org.hibernate.jdbc.AbstractBatcher.getResultSet(AbstractBatcher.java:215)
	at org.hibernate.loader.Loader.getResultSet(Loader.java:1805)
	at org.hibernate.loader.Loader.doQuery(Loader.java:697)
	at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:259)
	at org.hibernate.loader.Loader.doList(Loader.java:2228)
	... 18 more
2014-08-18 14:50:11,519 ERROR [Mina] - java.lang.Exception: org.hibernate.exception.ConstraintViolationException: could not execute query exception!
2014-08-18 14:50:12,626 ERROR [Mina] - New Connect VIN:
2014-08-18 14:50:12,642 ERROR [org.hibernate.util.JDBCExceptionReporter] - ORA-01400: cannot insert NULL into ("SFCV"."STATUS"."VIN")
ORA-06512: at "SFCV.J_PROCESSINFO", line 19
ORA-01403: no data found
ORA-06512: at line 1

2014-08-18 14:50:12,642 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.Exception: org.hibernate.exception.ConstraintViolationException: could not execute query
	at com.fcv.vms.model.HibernateModel.OnProcessReg(HibernateModel.java:242)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:96)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.hibernate.exception.ConstraintViolationException: could not execute query
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:94)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)
	at org.hibernate.loader.Loader.doList(Loader.java:2231)
	at org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2125)
	at org.hibernate.loader.Loader.list(Loader.java:2120)
	at org.hibernate.loader.custom.CustomLoader.list(CustomLoader.java:312)
	at org.hibernate.impl.SessionImpl.listCustomQuery(SessionImpl.java:1722)
	at org.hibernate.impl.AbstractSessionImpl.list(AbstractSessionImpl.java:165)
	at org.hibernate.impl.SQLQueryImpl.list(SQLQueryImpl.java:175)
	at com.fcv.vms.model.HibernateModel.OnProcessReg(HibernateModel.java:234)
	... 11 more
Caused by: java.sql.SQLException: ORA-01400: cannot insert NULL into ("SFCV"."STATUS"."VIN")
ORA-06512: at "SFCV.J_PROCESSINFO", line 19
ORA-01403: no data found
ORA-06512: at line 1

	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:331)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:288)
	at oracle.jdbc.driver.T4C8Oall.receive(T4C8Oall.java:745)
	at oracle.jdbc.driver.T4CCallableStatement.doOall8(T4CCallableStatement.java:215)
	at oracle.jdbc.driver.T4CCallableStatement.executeForRows(T4CCallableStatement.java:965)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1170)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3339)
	at oracle.jdbc.driver.OraclePreparedStatement.execute(OraclePreparedStatement.java:3445)
	at oracle.jdbc.driver.OracleCallableStatement.execute(OracleCallableStatement.java:4394)
	at com.mchange.v2.c3p0.impl.NewProxyCallableStatement.execute(NewProxyCallableStatement.java:3044)
	at org.hibernate.dialect.Oracle9Dialect.getResultSet(Oracle9Dialect.java:319)
	at org.hibernate.jdbc.AbstractBatcher.getResultSet(AbstractBatcher.java:215)
	at org.hibernate.loader.Loader.getResultSet(Loader.java:1805)
	at org.hibernate.loader.Loader.doQuery(Loader.java:697)
	at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:259)
	at org.hibernate.loader.Loader.doList(Loader.java:2228)
	... 18 more
2014-08-18 14:50:12,642 ERROR [Mina] -  is offline!
2014-08-18 14:50:12,642 ERROR [Mina] - java.lang.Exception: org.hibernate.exception.ConstraintViolationException: could not execute query exception!
2014-08-18 14:50:21,006 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 14:50:21,006 ERROR [Mina] - A0000038C1903EBV3 is offline!
2014-08-18 14:50:22,488 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:50:22,488 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:50:23,846 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:50:23,861 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:50:25,376 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:50:25,391 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 14:50:28,466 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 14:50:28,482 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:07:30,694 ERROR [Mina] - New Connect VIN:
2014-08-18 15:07:30,757 ERROR [org.hibernate.util.JDBCExceptionReporter] - ORA-01400: cannot insert NULL into ("SFCV"."STATUS"."VIN")
ORA-06512: at "SFCV.J_PROCESSINFO", line 19
ORA-01403: no data found
ORA-06512: at line 1

2014-08-18 15:07:30,757 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.Exception: org.hibernate.exception.ConstraintViolationException: could not execute query
	at com.fcv.vms.model.HibernateModel.OnProcessReg(HibernateModel.java:242)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:96)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.hibernate.exception.ConstraintViolationException: could not execute query
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:94)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)
	at org.hibernate.loader.Loader.doList(Loader.java:2231)
	at org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2125)
	at org.hibernate.loader.Loader.list(Loader.java:2120)
	at org.hibernate.loader.custom.CustomLoader.list(CustomLoader.java:312)
	at org.hibernate.impl.SessionImpl.listCustomQuery(SessionImpl.java:1722)
	at org.hibernate.impl.AbstractSessionImpl.list(AbstractSessionImpl.java:165)
	at org.hibernate.impl.SQLQueryImpl.list(SQLQueryImpl.java:175)
	at com.fcv.vms.model.HibernateModel.OnProcessReg(HibernateModel.java:234)
	... 11 more
Caused by: java.sql.SQLException: ORA-01400: cannot insert NULL into ("SFCV"."STATUS"."VIN")
ORA-06512: at "SFCV.J_PROCESSINFO", line 19
ORA-01403: no data found
ORA-06512: at line 1

	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:331)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:288)
	at oracle.jdbc.driver.T4C8Oall.receive(T4C8Oall.java:745)
	at oracle.jdbc.driver.T4CCallableStatement.doOall8(T4CCallableStatement.java:215)
	at oracle.jdbc.driver.T4CCallableStatement.executeForRows(T4CCallableStatement.java:965)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1170)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3339)
	at oracle.jdbc.driver.OraclePreparedStatement.execute(OraclePreparedStatement.java:3445)
	at oracle.jdbc.driver.OracleCallableStatement.execute(OracleCallableStatement.java:4394)
	at com.mchange.v2.c3p0.impl.NewProxyCallableStatement.execute(NewProxyCallableStatement.java:3044)
	at org.hibernate.dialect.Oracle9Dialect.getResultSet(Oracle9Dialect.java:319)
	at org.hibernate.jdbc.AbstractBatcher.getResultSet(AbstractBatcher.java:215)
	at org.hibernate.loader.Loader.getResultSet(Loader.java:1805)
	at org.hibernate.loader.Loader.doQuery(Loader.java:697)
	at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:259)
	at org.hibernate.loader.Loader.doList(Loader.java:2228)
	... 18 more
2014-08-18 15:07:30,772 ERROR [Mina] - java.lang.Exception: org.hibernate.exception.ConstraintViolationException: could not execute query exception!
2014-08-18 15:07:31,896 ERROR [Mina] - New Connect VIN:
2014-08-18 15:07:31,911 ERROR [org.hibernate.util.JDBCExceptionReporter] - ORA-01400: cannot insert NULL into ("SFCV"."STATUS"."VIN")
ORA-06512: at "SFCV.J_PROCESSINFO", line 19
ORA-01403: no data found
ORA-06512: at line 1

2014-08-18 15:07:31,911 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.Exception: org.hibernate.exception.ConstraintViolationException: could not execute query
	at com.fcv.vms.model.HibernateModel.OnProcessReg(HibernateModel.java:242)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:96)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.hibernate.exception.ConstraintViolationException: could not execute query
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:94)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)
	at org.hibernate.loader.Loader.doList(Loader.java:2231)
	at org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2125)
	at org.hibernate.loader.Loader.list(Loader.java:2120)
	at org.hibernate.loader.custom.CustomLoader.list(CustomLoader.java:312)
	at org.hibernate.impl.SessionImpl.listCustomQuery(SessionImpl.java:1722)
	at org.hibernate.impl.AbstractSessionImpl.list(AbstractSessionImpl.java:165)
	at org.hibernate.impl.SQLQueryImpl.list(SQLQueryImpl.java:175)
	at com.fcv.vms.model.HibernateModel.OnProcessReg(HibernateModel.java:234)
	... 11 more
Caused by: java.sql.SQLException: ORA-01400: cannot insert NULL into ("SFCV"."STATUS"."VIN")
ORA-06512: at "SFCV.J_PROCESSINFO", line 19
ORA-01403: no data found
ORA-06512: at line 1

	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:331)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:288)
	at oracle.jdbc.driver.T4C8Oall.receive(T4C8Oall.java:745)
	at oracle.jdbc.driver.T4CCallableStatement.doOall8(T4CCallableStatement.java:215)
	at oracle.jdbc.driver.T4CCallableStatement.executeForRows(T4CCallableStatement.java:965)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1170)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3339)
	at oracle.jdbc.driver.OraclePreparedStatement.execute(OraclePreparedStatement.java:3445)
	at oracle.jdbc.driver.OracleCallableStatement.execute(OracleCallableStatement.java:4394)
	at com.mchange.v2.c3p0.impl.NewProxyCallableStatement.execute(NewProxyCallableStatement.java:3044)
	at org.hibernate.dialect.Oracle9Dialect.getResultSet(Oracle9Dialect.java:319)
	at org.hibernate.jdbc.AbstractBatcher.getResultSet(AbstractBatcher.java:215)
	at org.hibernate.loader.Loader.getResultSet(Loader.java:1805)
	at org.hibernate.loader.Loader.doQuery(Loader.java:697)
	at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:259)
	at org.hibernate.loader.Loader.doList(Loader.java:2228)
	... 18 more
2014-08-18 15:07:31,911 ERROR [Mina] -  is offline!
2014-08-18 15:07:31,911 ERROR [Mina] - java.lang.Exception: org.hibernate.exception.ConstraintViolationException: could not execute query exception!
2014-08-18 15:07:40,166 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 15:07:42,725 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:07:42,725 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:07:44,161 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:07:44,161 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:07:45,285 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:07:45,301 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:07:46,581 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:07:46,581 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:07:50,469 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:07:50,484 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:07:53,763 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:07:53,779 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:07:55,965 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:07:55,980 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:08:00,352 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:08:00,368 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:08:03,630 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:08:03,646 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:08:09,142 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:08:09,158 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:08:10,251 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:08:10,266 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:08:13,545 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:08:13,561 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:08:16,934 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:08:16,949 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:08:19,025 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:08:19,041 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:08:23,444 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:08:23,459 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:08:26,832 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:08:26,848 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:08:28,971 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:08:28,987 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:08:33,343 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:08:33,359 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:08:36,653 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:08:36,669 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:08:42,150 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:08:42,165 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:08:43,243 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:08:43,258 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:08:46,553 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:08:46,568 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:08:49,862 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:08:49,878 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:08:52,048 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:08:52,064 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:08:56,482 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:08:56,498 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:08:59,746 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:08:59,761 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:09:01,948 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:09:01,963 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:09:06,335 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:09:06,351 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:10:56,804 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 15:17:50,670 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 15:17:53,338 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:17:53,338 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:17:54,774 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:102)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:17:54,774 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:33:30,699 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 15:33:32,150 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:103)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:33:32,150 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 15:33:35,474 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.StringIndexOutOfBoundsException: String index out of range: 5
	at java.lang.String.substring(Unknown Source)
	at com.fcv.vms.model.HibernateModel.OnSavedata(HibernateModel.java:46)
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:103)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 15:33:35,474 ERROR [Mina] - A0000038C1903EBV3java.lang.StringIndexOutOfBoundsException: String index out of range: 5 exception!
2014-08-18 16:06:22,706 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 16:06:24,173 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 231
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 16:06:24,173 ERROR [Mina] - A0000038C1903EBV3java.lang.ArrayIndexOutOfBoundsException: 231 exception!
2014-08-18 16:06:25,672 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 231
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 16:06:25,672 ERROR [Mina] - A0000038C1903EBV3java.lang.ArrayIndexOutOfBoundsException: 231 exception!
2014-08-18 16:37:00,390 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 16:37:00,405 ERROR [Mina] - A0000038C1903EBV3 is offline!
2014-08-18 16:37:01,873 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 231
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 16:37:01,888 ERROR [Mina] - A0000038C1903EBV3java.lang.ArrayIndexOutOfBoundsException: 231 exception!
2014-08-18 16:37:03,309 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 231
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 16:37:03,324 ERROR [Mina] - A0000038C1903EBV3java.lang.ArrayIndexOutOfBoundsException: 231 exception!
2014-08-18 16:37:27,868 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 231
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 16:37:27,868 ERROR [Mina] - A0000038C1903EBV3java.lang.ArrayIndexOutOfBoundsException: 231 exception!
2014-08-18 16:39:09,831 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 231
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 16:39:09,831 ERROR [Mina] - A0000038C1903EBV3java.lang.ArrayIndexOutOfBoundsException: 231 exception!
2014-08-18 16:39:51,853 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 16:39:51,853 ERROR [Mina] - A0000038C1903EBV3 is offline!
2014-08-18 16:39:53,304 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 231
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 16:39:53,320 ERROR [Mina] - A0000038C1903EBV3java.lang.ArrayIndexOutOfBoundsException: 231 exception!
2014-08-18 16:39:54,709 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 231
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 16:39:54,725 ERROR [Mina] - A0000038C1903EBV3java.lang.ArrayIndexOutOfBoundsException: 231 exception!
2014-08-18 16:43:27,621 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 16:43:29,118 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 231
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 16:43:29,118 ERROR [Mina] - A0000038C1903EBV3java.lang.ArrayIndexOutOfBoundsException: 231 exception!
2014-08-18 16:43:30,586 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 226
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 16:43:30,586 ERROR [Mina] - A0000038C1903EBV3java.lang.ArrayIndexOutOfBoundsException: 226 exception!
2014-08-18 16:50:06,829 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 16:50:08,295 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 231
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 16:50:08,295 ERROR [Mina] - A0000038C1903EBV3java.lang.ArrayIndexOutOfBoundsException: 231 exception!
2014-08-18 16:50:09,248 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 226
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 16:50:09,248 ERROR [Mina] - A0000038C1903EBV3java.lang.ArrayIndexOutOfBoundsException: 226 exception!
2014-08-18 17:00:00,815 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 17:00:02,390 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 231
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 17:00:02,390 ERROR [Mina] - A0000038C1903EBV3java.lang.ArrayIndexOutOfBoundsException: 231 exception!
2014-08-18 17:00:03,343 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 226
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 17:00:03,343 ERROR [Mina] - A0000038C1903EBV3java.lang.ArrayIndexOutOfBoundsException: 226 exception!
2014-08-18 17:04:58,460 ERROR [Mina] - A0000038C1903EBV3 is offline!
2014-08-18 17:04:58,460 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 17:05:01,035 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 226
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 17:05:01,050 ERROR [Mina] - A0000038C1903EBV3java.lang.ArrayIndexOutOfBoundsException: 226 exception!
2014-08-18 17:05:02,502 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 226
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 17:05:02,502 ERROR [Mina] - A0000038C1903EBV3java.lang.ArrayIndexOutOfBoundsException: 226 exception!
2014-08-18 17:05:03,533 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 231
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 17:05:03,548 ERROR [Mina] - A0000038C1903EBV3java.lang.ArrayIndexOutOfBoundsException: 231 exception!
2014-08-18 17:14:41,183 ERROR [Mina] - A0000038C1903EBV3:both idle timeout!
2014-08-18 17:14:41,183 ERROR [Mina] - A0000038C1903EBV3 is offline!
2014-08-18 17:44:28,651 ERROR [Mina] - New Connect VIN:A0000038C1903EBV3
2014-08-18 17:44:31,210 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 226
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 17:44:31,226 ERROR [Mina] - A0000038C1903EBV3java.lang.ArrayIndexOutOfBoundsException: 226 exception!
2014-08-18 17:44:32,646 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 226
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 17:44:32,646 ERROR [Mina] - A0000038C1903EBV3java.lang.ArrayIndexOutOfBoundsException: 226 exception!
2014-08-18 17:44:33,552 ERROR [org.apache.mina.filter.logging.LoggingFilter] - EXCEPTION :
java.lang.ArrayIndexOutOfBoundsException: 231
	at com.fcv.vms.server.MyHandler.messageReceived(MyHandler.java:75)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$TailFilter.messageReceived(DefaultIoFilterChain.java:722)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.callNextMessageReceived(DefaultIoFilterChain.java:434)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain.access$1200(DefaultIoFilterChain.java:48)
	at org.apache.mina.core.filterchain.DefaultIoFilterChain$EntryImpl$1.messageReceived(DefaultIoFilterChain.java:802)
	at org.apache.mina.core.filterchain.IoFilterEvent.fire(IoFilterEvent.java:59)
	at org.apache.mina.core.session.IoEvent.run(IoEvent.java:64)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTask(OrderedThreadPoolExecutor.java:552)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.runTasks(OrderedThreadPoolExecutor.java:544)
	at org.apache.mina.filter.executor.OrderedThreadPoolExecutor$Worker.run(OrderedThreadPoolExecutor.java:488)
	at java.lang.Thread.run(Unknown Source)
2014-08-18 17:44:33,552 ERROR [Mina] - A0000038C1903EBV3java.lang.ArrayIndexOutOfBoundsException: 231 exception!
2014-08-18 17:52:21,905 ERROR [Mina] - A0000038C1903EBV3:both idle timeout!
2014-08-18 17:52:21,905 ERROR [Mina] - A0000038C1903EBV3 is offline!
